test/serialisers/json-0.6.js

```diff
@@ -483,5 +506,63 @@ describe('JSON Serialiser', function() {
       expect(element).to.be.instanceof(minim.elements.Array);
       expect(element.get(0)).to.be.instanceof(minim.elements.Number);
     });
+
+    it('deserialises enum', function() {
```

Comment:
Deserialising the 0.6 format into how the enum element works in API Elements 1.0.